src/WebApi/Controllers/ServicesController.cs

```diff
@@ -1,29 +1,62 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using AutoMapper;
+using BusinessLayer;
+using BusinessLayer.Contracts;
+using JetBrains.Annotations;
 using Microsoft.AspNetCore.Mvc;
-using WebApi.Models.Service;
+using ApiServiceModel = WebApi.Models.Service.ServiceModel;
+using BlServiceModel = BusinessLayer.Models.ServiceModel;
 
 namespace WebApi.Controllers
 {
     [Route("api/[controller]")]
     public class ServicesController : Controller
     {
+        [NotNull]
+        private readonly IServiceService _serviceService;
+
+        public ServicesController([NotNull] IServiceService serviceService)
+        {
+            _serviceService = serviceService;
+        }
+
         // GET /services
         [HttpGet]
-        public IEnumerable<ServiceModelResponse> Get(){
-            List<ServiceModelResponse> services = new List<ServiceModelResponse>();
-            return services;
+        public async Task<IActionResult> Get()
+        {
+            IEnumerable<BlServiceModel> services = await _serviceService.GetServices();
+
+            return Ok(
+                services.Select(Mapper.Map<ApiServiceModel>)
+            );
         }
 
-        // PUT /services
-        [HttpPost]
-        public IActionResult Put(ServiceModelRequest service){
-            return StatusCode((int)System.Net.HttpStatusCode.Created);
+        // PUT /services -> add or update service
+        [HttpPut]
+        public async Task<IActionResult> Put([FromBody]ApiServiceModel service)
+        {
+            BlServiceModel newService = await _serviceService.AddOrUpdateService(Mapper.Map<BlServiceModel>(service));
+
+            return Ok(
+                Mapper.Map<ApiServiceModel>(newService)
+            );
         }
 
         // DELETE /services/{id}
         [HttpDelete("{id:int}")]
-        public IActionResult Delete(int id){
-            return StatusCode((int)System.Net.HttpStatusCode.Accepted);
+        public async Task<IActionResult> Delete(int id)
+        {
+             StoredProcedureExecutionResult result =  await _serviceService.DeleteService(id);
+
+            if (result == StoredProcedureExecutionResult.Ok)
+            {
+                return Ok();
+            }
+
+            return BadRequest(HttpStatusCode.NotAcceptable);
```

Comment:
I recommend to check for bad cases and return bad status in ifs. At the end return Ok status. In general here should be returned 202 (accepted) instead of 200 OK and bad request should return 409 Conflict instead of NotAcceptable